pkg/cmd/run.go

```diff
@@ -117,6 +117,7 @@ func newCmdRun(rootCmdOptions *RootCmdOptions) (*cobra.Command, *runCmdOptions)
 	cmd.Flags().String("pod-template", "", "The path of the YAML file containing a PodSpec template to be used for the Integration pods")
 
 	cmd.Flags().Bool("save", false, "Save the run parameters into the default kamel configuration file (kamel-config.yaml)")
+	cmd.Flags().Bool("verbose", false, "Verbose logging")
```

Comment:
Is there no shorthand name for the flag? :smiley:

It looks like the `verbose` flag can be global, but maybe we can work on it later?

Reply:
Ah yeah b/c it was already taken by the volume flag ... wasn't sure what to do in that case :) vb ?

Reply:
Ah I see, we already have `--volume`... Maybe then we can use `-V`?
It's ok to leave it without a shorthand flag if we don't find a good one.